packages/app/obojobo-repository/shared/components/pages/page-dashboard-client.jsx

```diff
@@ -1,8 +1,10 @@
-import { hydrateEl } from '../../react-utils'
+import { hydrateEl, hydrateElWithoutStore } from '../../react-utils'
```

Comment:
Since the notification component is no longer being rendered/hydrated, `hydrateElWithoutStore` is no longer a necessary import here.